dns-common/on_boot.d/10-dns.sh

```diff
@@ -71,7 +75,8 @@ fi
 
 # Make DNSMasq listen to the container network for split horizon or conditional forwarding
 if ! grep -qxF interface=br$VLAN.mac /run/dnsmasq.conf.d/custom.conf; then
-    echo interface=br$VLAN.mac >> /run/dnsmasq.conf.d/custom.conf
```

Comment:
Can you expound on the removal of the `echo` command here? (Minor, but the added line break can probably be removed as well.)

Reply:
oops, sorry the removal of echo was my own fat fingeredness :)

Reply:
Ah, glad I commented. 
+1 for code review! New commit coming? ;)